include/ssd-internal.h

```diff
@@ -2,6 +2,7 @@
 #ifndef __LABWC_SSD_INTERNAL_H
 #define __LABWC_SSD_INTERNAL_H
 
+#include <wlr/util/box.h>
```

Comment:
Do we actually need this?
A forward declaration should be enough if all users of this header include box.h themselves.

Reply:
I think the `struct wlr_box geometry;` line is a compile error (if the header were to be compiled on its own) without this.

Reply:
Not sure what you mean by "compiled on its on" but in any case the same would be true for e.g. `struct view` or `struct wlr_scene_node` then as they are only forward-declared in `ssd.h` as well (which is included by `ssd-internal.h`).